tests/stdlib/tjsonutils.nim

```diff
@@ -13,7 +13,16 @@ proc testRoundtrip[T](t: T, expected: string) =
   t2.fromJson(j)
   doAssert t2.toJson == j
 
-import tables
+template expect(Error: type, body: untyped) =
```

Comment:
does `doAssertRaises` not work here?

Reply:
I didn't know about the existence of `doAssertRaises`. I will fix it.

Reply:
Done.

Reply:
(feel free to click "resolve conversation" when you resolve and there's no pushback)